test/browser.core.test.js

```diff
@@ -43,23 +43,26 @@ describe('options.captureUncaught', function() {
 
     var element = document.getElementById('throw-error');
     element.click();
-    server.respond();
 
-    var body = JSON.parse(server.requests[0].requestBody);
+    setTimeout(function() {
```

Comment:
__*Comment*__
This strategy should work find 99% of the time, just watch out for flakiness. I don't think that there is a guarantee on execution order based on the length of the timeout

Reply:
True, there's no guarantee.